app/controllers/defendants_controller.rb

```diff
@@ -1,33 +1,27 @@
 # frozen_string_literal: true
 
 class DefendantsController < ApplicationController
-  include DefendantSearchable
-
-  before_action :load_and_authorize_search
+  before_action :authorize_defendant_search
```

Comment:
this should be called `load_authorize_defendant_search`

Reply:
👍 amended to load_and_authorize_defendant_search